CHANGELOG.md

```diff
@@ -63,9 +64,4 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ### Security
 
-<<<<<<< HEAD
```

Comment:
@kotwanikunal whichever wins the flaky tests lottery first ...

Reply:
Hah. :) 
I'll close the other one since this achieves more than just the fix